### ISSUE TITLE
Aria2 Status: Not connected: How to troubleshoot ?

### ISSUE THREAD

**Issue by skarfr** [mayswind/AriaNg#53]
Hi,

Thank you for AriaNg. I installed it and somehow, i failed to make it work.
I have setup my aria2 rpc server and it is working (i can connect using a client on my computer).

Using AriaNg Settings, i try the same parameters (my public IP and my RPC Secret Token). I refresh (to save) and I still get "Not Connected".

Question: Is there a log file or anything else to help me troubleshoot my issue ?

Thanks again for AriaNg and thanks also for your help

**Comment by mayswind** [comment]
hi~
Most browsers have a Developer Tools. For example, in Chrome, you can press Ctrl+Shift+J in Windows or Command+Option+J in macOS.
![chrome](https://cloud.githubusercontent.com/assets/2211648/26361451/97541cda-400d-11e7-8f07-18a1f45518ca.png)

**Comment by skarfr** [comment]
Ooo I see, i didn't think about checking the browser console.

Thanks to your advice, i found out what my problem was: a "CORS" issue:
`XMLHttpRequest cannot load https://mydomain:6800/jsonrpc. Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://mydomain' is therefore not allowed access.`

I solved it using the aria2c option: rpc-allow-origin-all=true

This is now solved, thanks
